NAMESPACE

```diff
@@ -7,7 +7,6 @@ export(abline)
 export(arrows)
 export(axis)
 export(bgCol)
-export(bgCol.default)
```

Comment:
Why did this get deleted? And, if that works, can we lose our other custom function's .default exports? (error_bars, callouts)